src/api/forms/service.js

```diff
@@ -1,25 +1,140 @@
+import { readFile } from 'node:fs/promises'
+
+import { Schema } from '@defra/forms-model'
+
+import { createLogger } from './../../helpers/logging/logger.js'
+import {
+  FailedCreationOperationError,
+  FormAlreadyExistsError,
+  InvalidFormDefinitionError
+} from './errors.js'
+import {
+  createFormDefinition,
+  getFormDefinition as getFormDefinitionFromRepository
+} from './form-definition-repository.js'
 import {
+  createFormMetadata,
   listForms as listFormMetadataEntries,
+  exists as formMetadataExists,
   getFormMetadata
 } from './form-metadata-repository.js'
```

Comment:
Worth keeping this as a namespace import?

```js
metadata.create()
metadata.exists()
metadata.get()
metadata.getById()
```

```suggestion
import * as metadata from './form-metadata-repository.js'
```